src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/standard_constructions.jl

```diff
@@ -491,6 +491,49 @@ end
 
 
 
+@doc raw"""
+    normal_toric_varieties_from_star_triangulations(P::Polyhedron; set_attributes::Bool = true)
```

Comment:
The function name at the start of the docstring here doesn't seem to match that actual function name (pluralization)

Reply:
Thx for spotting this. Just pushed an update which (should) fix this.